staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1690,10 +1690,12 @@ message WindowsMemoryUsage {
     int64 timestamp = 1;
     // The amount of working set memory in bytes.
     UInt64Value working_set_bytes = 2;
-    // Available memory for use. This is defined as the memory limit - workingSetBytes.
+    // Available memory for use. This is defined as the memory limit - commit_memory_bytes.
```

Comment:
if you want to add the additional note about past behavior, please do it. I lgtm either way, just unhold of you do not plan to add a note

/hold

Reply:
I have the changes for exposing this field from hcsshim and containerd already. Once this is checked in and tag for 1.28 is cut, I'll send a PR for them immediately.

/unhold